src/utils/validators.js

```diff
@@ -169,329 +169,304 @@ export const MEDIA_SETTINGS_TITLE_MIN_LENGTH = 6
 export const MEDIA_SETTINGS_TITLE_MAX_LENGTH = 100
 export const MEDIA_FILE_MAX_SIZE = 5242880 // 5MB -> 5242880B
 
+const validateAndSetSectionError = (validator) => (
+  sectionError,
+  sectionType,
+  field,
+  value
+) => {
+  const newSectionError = sectionError
+  const errorMessage = validator(field, value)
+  newSectionError[sectionType][field] = errorMessage
+  return newSectionError
+}
+
 // Homepage Editor
 // ==========
-// Returns new errors.highlights[index] object
-const validateHighlights = (highlightError, field, value) => {
-  const newHighlightError = highlightError
-  let errorMessage = ""
+const validateHighlight = (field, value) => {
   switch (field) {
     case "title": {
       // Title is too short
-      if (value.length < HIGHLIGHTS_TITLE_MIN_LENGTH) {
-        errorMessage = `The title should be longer than ${HIGHLIGHTS_TITLE_MIN_LENGTH} characters.`
+      if (value.length <= HIGHLIGHTS_TITLE_MIN_LENGTH) {
+        return `The title should be longer than ${HIGHLIGHTS_TITLE_MIN_LENGTH} characters.`
       }
       // Title is too long
       if (value.length > HIGHLIGHTS_TITLE_MAX_LENGTH) {
-        errorMessage = `The title should be shorter than ${HIGHLIGHTS_TITLE_MAX_LENGTH} characters.`
+        return `The title should be shorter than ${HIGHLIGHTS_TITLE_MAX_LENGTH} characters.`
```

Comment:
The max length comparisons on this page should be `>=` to match the error message!

Reply:
lmao good catch - 4 years in the making HAHAHAHA